org/mozilla/jss/tests/all.pl

```diff
@@ -317,11 +317,14 @@ sub outputEnv {
 
    print "*****ENVIRONMENT*****\n";
    print "java=$java\n";
-   print "NATIVE_FLAG=$ENV{NATIVE_FLAG}\n";
    print "$ld_lib_path=$ENV{$ld_lib_path}\n";
-   print "CLASSPATH=$ENV{CLASSPATH}\n";
-   print "BUILD_OPT=$ENV{BUILD_OPT}\n";
-   print "USE_64=$ENV{USE_64}\n";
+   print "CLASSPATH=$classpath\n";
+   if (defined $ENV{BUILD_OPT}) {
```

Comment:
I wasn't sure the purpose of printing this at the test stage, but figured it'd be OK to leave it in.